coding-concepts.js

```diff
@@ -50,8 +51,8 @@ const myCodingSkills = {
 }
 // console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: nothing
+// b) Verify and explain: to invoke an object, you have to console.log(myCodingSkills.languages) but to get the index of the object you do something that I do not know how to do. 
```

Comment:
Remember that to access key value pairs in an object, we can use dot notation. So when we use .languages on the myCodingSkills object like so: `myCodingSkills.languages`, we are accessing the value at the key of `languages`. Then, we are using bracket notation on that value to indicate we want the value at index 0, which is the string `"Javascript"`